ui.frontend/src/LanguageUtils.js

```diff
@@ -94,6 +94,9 @@ class LanguageUtils {
                         ? snippets[number]
                         : match;
                 });
+
+                //  Remove all the $ symbols in alet message
+                     translatedText = translatedText .replace(/\$/g, '');   
```

Comment:
Can we change the main regex to `/\${(\d+)}/g`

Reply:
Done!